### PR TITLE
Fix losing the position packet when teleporting

### PR DESCRIPTION
**Pull request by PatoTheBest**

### CODE REVIEW

File: src/protocolsupport/protocol/packet/middle/serverbound/play/MiddlePositionLook.java [hunk 1]
```diff
@@ -20,17 +21,24 @@
...
+				collection.add(MiddleTeleportAccept.createPacket(teleportId));
+				collection.add(createPacket());
+				return collection;
 			}
 		}
+
+		return RecyclableSingletonList.create(createPacket());
+	}
+
+	private ServerBoundPacketData createPacket() {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middle/serverbound/play/MiddlePositionLook.java]: Inline

Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middle/serverbound/play/MiddlePositionLook.java]: createPacket -> createMoveLookPacket

File: src/protocolsupport/protocol/packet/middle/serverbound/play/MiddleTeleportAccept.java [hunk 2]
```diff
@@ -21,10 +21,13 @@
 		}
 	}
 
-	public static RecyclableCollection<ServerBoundPacketData> create(int teleportId) {
+	private static RecyclableCollection<ServerBoundPacketData> create(int teleportId) {
+		return RecyclableSingletonList.create(createPacket(teleportId));
+	}
+
+	public static ServerBoundPacketData createPacket(int teleportId) {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middle/serverbound/play/MiddleTeleportAccept.java]: createPacket -> create

File: src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_4_5/PositionLook.java [hunk 3]
```diff
@@ -53,10 +54,23 @@ public void readFromClientData(ByteBuf clientdata) {
...
+					RecyclableCollection<ServerBoundPacketData> collection = RecyclableArrayList.create();
+					collection.add(MiddleTeleportAccept.createPacket(teleportId));
+					collection.add(createPacket());
+					return collection;
 				}
 			}
 		}
 	}
 
+	private ServerBoundPacketData createPacket() {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_4_5/PositionLook.java]: createPacket -> createMoveLookPacket

**Comment by PatoTheBest** [comment]
Should I squash the commits?

**Comment by Shevchik** [comment]
No

**Pull request merged**